volttron/platform/agent/base_historian.py

```diff
@@ -304,9 +304,9 @@ def _create_subscriptions(self):
         subscriptions = [
             (topics.DRIVER_TOPIC_BASE, self._capture_device_data),
             (topics.LOGGER_BASE, self._capture_log_data),
-            (topics.ACTUATOR, self._capture_actuator_data),
+            #(topics.ACTUATOR, self._capture_actuator_data),
```

Comment:
Yeah, we're just going to punt for now. We need some sort of basehistorian spec. I was kind of winging it when I wrote it the first time.